### PR TITLE
added general evaluator documentation

### PR DESCRIPTION
**Pull request by Gerwoud**

### CODE REVIEW

File: documentation/docs/evaluators/general_evaluator.md [hunk 1]
```diff
@@ -1 +1,8 @@
-# General evaluator
\ No newline at end of file
+# General evaluator
+The general runner is capable of running the standard language of the following languages
+- Javascript
+- Python
+- Ruby
+- Java
+
+All the user had to is upload a run_tests.sh in the project files and the script will run when a submissions is uploaded by a student.
```
Comment (Vucis) [inline documentation/docs/evaluators/general_evaluator.md:8]: had -> has
submissions -> submission

File: documentation/docs/evaluators/general_evaluator.md [hunk 2]
```diff
@@ -1 +1,8 @@
-# General evaluator
\ No newline at end of file
+# General evaluator
+The general runner is capable of running the standard language of the following languages
```
Comment (Vucis) [inline documentation/docs/evaluators/general_evaluator.md:2]: what do you mean with the standard language?

**Review by Vucis — CHANGES_REQUESTED** [review]

**Review by AronBuzogany — APPROVED** [review]

File: documentation/docs/evaluators/general_evaluator.md (hunk 2 above)
Comment (Gerwoud) [reply documentation/docs/evaluators/general_evaluator.md:2]: The language without any extra installations

Comment (Gerwoud) [reply documentation/docs/evaluators/general_evaluator.md:2]: Maybe base language better?

File: documentation/docs/evaluators/general_evaluator.md (hunk 1 above)
Comment (Gerwoud) [reply documentation/docs/evaluators/general_evaluator.md]: https://github.com/SELab-2/UGent-3/pull/424/commits/3cfa3f4613517029fff5986ea5033d34a77f69dc

File: documentation/docs/evaluators/general_evaluator.md (hunk 2 above)
Comment (Gerwoud) [reply documentation/docs/evaluators/general_evaluator.md]: https://github.com/SELab-2/UGent-3/pull/424/commits/691b8301650b9c917c2b1a5a893714ae1d2ff1f6

File: documentation/docs/evaluators/general_evaluator.md [hunk 3]
```diff
@@ -1 +1,8 @@
-# General evaluator
\ No newline at end of file
+# General evaluator
+The general runner is capable of running the standard library of the following languages
```
Comment (Vucis) [inline documentation/docs/evaluators/general_evaluator.md:2]: running the following languages with standard libraries included

File: documentation/i18n/nl/docusaurus-plugin-content-docs/current/evaluators/general_evaluator.md [hunk 4]
```diff
@@ -1 +1,8 @@
-# General evaluator
\ No newline at end of file
+# General evaluator
+De algemene runner is in staat om de standaardbibliotheek van de volgende talen uit te voeren:
```
Comment (Vucis) [inline documentation/i18n/nl/docusaurus-plugin-content-docs/current/evaluators/general_evaluator.md:2]: same here

**Review by Vucis — CHANGES_REQUESTED** [review]

File: documentation/i18n/nl/docusaurus-plugin-content-docs/current/evaluators/general_evaluator.md (hunk 4 above)
Comment (Gerwoud) [reply documentation/i18n/nl/docusaurus-plugin-content-docs/current/evaluators/general_evaluator.md]: https://github.com/SELab-2/UGent-3/pull/424/commits/707631c951bb7f16d4e73d19aba62882d8f974ed

File: documentation/docs/evaluators/general_evaluator.md (hunk 3 above)
Comment (Gerwoud) [reply documentation/docs/evaluators/general_evaluator.md]: https://github.com/SELab-2/UGent-3/pull/424/commits/707631c951bb7f16d4e73d19aba62882d8f974ed

File: documentation/i18n/nl/docusaurus-plugin-content-docs/current/evaluators/general_evaluator.md [hunk 5]
```diff
@@ -1 +1,8 @@
-# General evaluator
\ No newline at end of file
+# General evaluator
+De algemene runner is in staat om de volgende talen uit te voeren met enkel de standaardbibliotheek
```
Comment (Vucis) [inline documentation/i18n/nl/docusaurus-plugin-content-docs/current/evaluators/general_evaluator.md:2]: standaardbibliotheek -> standaardbibliotheken inbegrepen

**Review by Vucis — CHANGES_REQUESTED** [review]

File: documentation/i18n/nl/docusaurus-plugin-content-docs/current/evaluators/general_evaluator.md (hunk 5 above)
Comment (Gerwoud) [reply documentation/i18n/nl/docusaurus-plugin-content-docs/current/evaluators/general_evaluator.md]: https://github.com/SELab-2/UGent-3/pull/424/commits/b7ac124b4f3c2286e5e1a2b81e40ba2c6cce2299

**Review by Vucis — APPROVED** [review]
Final fix, pop off

**Pull request merged**